docs/lineage.rst

```diff
@@ -0,0 +1,87 @@
+Lineage
```

Comment:
Maybe mention it is still in alpha?

Reply:
It will be in master not in a release yet